charts/prometheus-node-exporter/values.yaml

```diff
@@ -395,6 +407,9 @@ extraHostVolumeMounts: []
 #    mountPath: <mountPath>
 #    readOnly: true|false
 #    mountPropagation: None|HostToContainer|Bidirectional
+#    https://kubernetes.io/docs/concepts/storage/volumes/#hostpath-volume-types
+#    type: "" (Default)|DirectoryOrCreate|Directory|FileOrCreate|File|Socket|CharDevice|BlockDevice
```

Comment:
Although set here, the change does not introduce `type` in `volumes.hostPath`.

Reply:
I moved this var to be set in `volumes.hostPath`